rollup.config.js

```diff
@@ -21,7 +21,7 @@ export default [
 		},
 	},
 	{
-		input: "src/index.umd.js",
+		input: "src/Hammer.js",
```

Comment:
@younkue Doesn't it need to apply in minified target? and Is `index.umd.js` useless anymore?

Reply:
@jongmoon  Hammer.js is same as index.umd.js. removed index.umd.js